src/main/java/com/semmle/jira/addon/util/JiraUtils.java

```diff
@@ -0,0 +1,108 @@
+package com.semmle.jira.addon.util;
+
+import com.atlassian.jira.component.ComponentAccessor;
+import com.atlassian.jira.issue.fields.config.FieldConfigScheme;
+import com.atlassian.jira.issue.fields.config.manager.IssueTypeSchemeManager;
+import com.atlassian.jira.issue.issuetype.IssueType;
+import com.atlassian.jira.issue.status.Status;
+import com.atlassian.jira.project.Project;
+import com.atlassian.jira.workflow.JiraWorkflow;
+import com.atlassian.jira.workflow.WorkflowManager;
+import com.atlassian.jira.workflow.WorkflowSchemeManager;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.ofbiz.core.entity.GenericEntityException;
+import org.ofbiz.core.entity.GenericValue;
+
+public class JiraUtils {
+
+  public static void addIssueTypeToProject(String projectKey, String issueTypeName)
```

Comment:
I think this method should take a project ID as argument. That way you can move the `getProjectByCurrentKey` call out of this method. Same for issue type.